### ISSUE TITLE
upgrade to latest axe-core

### ISSUE THREAD

**Issue by ctcpip** [ember-a11y/ember-a11y-testing#79]
There are fixes to known issues in the new versions of axe-core.  Can we upgrade to the latest?  (2.2.1 at time of writing)

**Comment by ctcpip** [comment]
```sh
$ npm outdated
Package                               Current  Wanted        Latest  Location
axe-core                                2.2.1   2.1.7         2.2.1  ember-a11y-testing
```

**Comment by trentmwillis** [comment]
I just published version `0.4.2` which relaxes the semver range for axe-core, so you should be able to pick up the latest!
